x-pack/plugins/canvas/server/routes/custom_elements/create.test.ts

```diff
@@ -43,7 +38,7 @@ describe('POST custom element', () => {
 
     const httpService = httpServiceMock.createSetupContract();
 
-    const router = httpService.createRouter('') as jest.Mocked<IRouter>;
+    const router = httpService.createRouter();
```

Comment:
As `createSetupContract` was returning the internal contract, the `createRouter` signature was the internal one. Removed the now invalid `path` parameter in plugins test files.